File: packages/binding/src/bindings/python/wrap.py
class Primate():
    @staticmethod
    def view(name, props = None, options = None):
        return {
            "__handler__": "view",
            "name": name,
            "props": props,
            "options": options,
        }
    @staticmethod
    def redirect(location, options = None):
        return {
            "__handler__": "redirect",
            "options": options,
        }
    @staticmethod
    def error(body, options = None):
        return {
            "__handler__": "error",
            "body": body,
            "options": options,
        }

File: packages/python/src/wrap.ts
export default (code: string) => `
class Primate():
    @staticmethod
    def view(name, props = None, options = None):
        return {
            "handler": "view",
            "params": (name, props, options),
        }
    @staticmethod
    def redirect(location, options = None):
        return {
            "handler": "redirect",
            "params": (location, options),
        }
    @staticmethod
    def error(body, options = None):
        return {
            "handler": "error",
            "params": (body, options),
        }\n${code}`;
